trunk/NDHMS/Routing/module_NWM_io_dict.F

```diff
@@ -510,6 +530,9 @@ module module_NWM_io_dict
    character (len=64), dimension(numGwVars) :: varNames
    integer :: numVars = numGwVars
    character (len=512) :: outVersion = codeVersion
+   character (len=512) :: modelOutputType = "groundWater_rt"
```

Comment:
Super minor thing here, but my vote would be for "groundwater_rt" since USGS + the scientific community made a rather large point of saying that groundwater should now be one word back in 2009 or so.  Memos for reference, context, and humor:

https://water.usgs.gov/admin/memo/GW/gw75.03.html
https://water.usgs.gov/admin/memo/GW/gw09.03.html

And in the journal Groundwater (though it may have been Ground Water at the time ;) ):
https://doi.org/10.1111/j.1745-6584.2009.00566.x

Reply:
Good comment! I did not know that. I will fix this.